bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieImpl.java

```diff
@@ -563,7 +563,7 @@ public static BookieSocketAddress getBookieAddress(ServerConfiguration conf)
             && !conf.getAllowLoopback()) {
             throw new UnknownHostException("Trying to listen on loopback address, "
                     + addr + " but this is forbidden by default "
-                    + "(see ServerConfiguration#getAllowLoopback()).\n"
+                    + "(see ServerConfiguration#getAllowLoopback())." + System.lineSeparator()
```

Comment:
why?
This is an Exception message, probably it is better to remove the '\n' character